website/docs/getting-started/setup/upgrade-to-business-edition/README.md

```diff
@@ -1,13 +1,18 @@
 ---
-description: Follow the steps to upgrade the Appsmith installation to a Business Edition.
+description: Follow these steps to upgrade the Appsmith Community Edition to a Business Edition.
 ---
 # Upgrade to Business Edition
 
-:::info Important
-You need a **License Key** to complete the upgrade. Learn more on [Business Edition & pricing](https://www.appsmith.com/pricing) options.
-:::
+The Appsmith Business Edition provides a suite of advanced features and capabilities designed to support the needs of your growing organization:
 
-Upgrade your Community Edition to a Business Edition by selecting below installation type.
+* Single Sign On (SSO) using Okta, Auth0, Azure AD + any SAML, OIDC SSO
+* Enhanced Access control features like Granular Access Control
+* Team collaboration tools like Version Control with Git
```

Comment:
remove this as this is available in community except the part for more than 3 private repos